### PR TITLE
Set darwin's eglot config dir

### PR DESCRIPTION
**Pull request by xiliuya**

### CODE REVIEW

**Review by jcs090218 — APPROVED** [review]

**Comment by bbbscarter** [comment]
Just to clarify - this is a separate problem, and I don't think it fixes the main bit of #137.  

Even with the correct port number, eglot disconnects.

**Comment by NathanLovato** [comment]
Thanks for your work. Do you not have merge permissions? Don't hesitate to merge your own PRs once they're approved.

**Pull request merged**
